alkira/resource_alkira_policy_nat.go

```diff
@@ -65,6 +65,12 @@ func resourceAlkiraPolicyNat() *schema.Resource {
 				Elem:        &schema.Schema{Type: schema.TypeInt},
 				Required:    true,
 			},
+			"category": {
```

Comment:
Do we have a default value for this?

Reply:
There isn't a default value, but an empty value is treated the same as `DEFAULT`. I've updated the description to make this clear